### PR TITLE
docs: working ollama for desktop

### PR DESCRIPTION
**Pull request by michaelneale**
cc @codefromthecrypt

### CODE REVIEW

File: packages/exchange/src/exchange/providers/ollama.py [hunk 1]
```diff
@@ -16,14 +16,14 @@ class OllamaProvider(OpenAiProvider):
...
-      provider: ollama
-      processor: {OLLAMA_MODEL}
-      accelerator: {OLLAMA_MODEL}
-      moderator: passive
-      toolkits:
-      - name: developer
-        requires: {{}}
+ollama:
+  provider: ollama
+  processor: qwen2
```
Comment (codefromthecrypt) [inline packages/exchange/src/exchange/providers/ollama.py]: qwen2.5? Also should we not change OLLAMA_MODEL to that?

Comment (michaelneale) [reply packages/exchange/src/exchange/providers/ollama.py]: testing it now

Comment (michaelneale) [reply packages/exchange/src/exchange/providers/ollama.py]: @codefromthecrypt will it change CI behavior? so far qwen 2.5 seems quite good

File: packages/exchange/src/exchange/providers/ollama.py [hunk 2]
```diff
@@ -12,18 +12,18 @@
...
-
-    ollama:
-      provider: ollama
-      processor: {OLLAMA_MODEL}
-      accelerator: {OLLAMA_MODEL}
-      moderator: passive
-      toolkits:
-      - name: developer
-        requires: {{}}
+    __doc__ += f"""Here's an example profile configuration to try:
```
Comment (baxen) [inline packages/exchange/src/exchange/providers/ollama.py]: need to remove the "f" here, its causing an error when it tries to interpret the {} below as a formatter

**Review by baxen — APPROVED** [review]
LGTM

File: packages/exchange/src/exchange/providers/ollama.py (hunk 1 above)
Comment (codefromthecrypt) [reply packages/exchange/src/exchange/providers/ollama.py]: sorry I was AWOL last week. CI tests with qwen2.5, so will follow-up with a change to update the constant

Comment (codefromthecrypt) [reply packages/exchange/src/exchange/providers/ollama.py]: https://github.com/block-open-source/goose/pull/150

**Pull request merged**